src/Applications/LDAS_App/ldas_setup

```diff
@@ -194,6 +196,18 @@ class LDASsetup:
                                                _datediff.minutes,
                                                _datediff.seconds)
             self.job_sgmt.append("JOB_SGMT: "+self.rqdExeInp['JOB_SGMT'])
+
+        if 'NUM_SGMT' not in self.rqdExeInp:
+           self.rqdExeInp['NUM_SGMT'] = 1
+        # hard set NUM_SGMT and NUM_SGMT
+        if (self.ladas_coupling > 0) :
+           if self.rqdExeInp['NUM_SGMT'] != 1 :
+              print "WARNING: 'NUM_SGMT' is reset to 1"
+              self.rqdExeInp['NUM_SGMT'] = 1 
```

Comment:
I don't think this is consistent with my earlier comment:
> Exit setup with message that clarifies this.
It's probably better that ldas_setup exits at this point.  The user can then change his/her setting accordingly, rather than have ldas_setup hardwire something that happens (very likely) without the user noticing.  (Warnings are easy to miss.)